### ISSUE TITLE
NIP-07 authentication

### ISSUE THREAD

**Issue by itstomekk** [untreu2/nosweet#1]
Hey, nice project!

Can you please implement NIP-07 login with the extension like Alby, so users do not have to copypaste their nsecs?

Thank you!

**Comment by untreu2** [comment]
of course, i'll implement it this week. can you tell me where you found my website?

**Comment by itstomekk** [comment]
I'm hunting for such new projects ;)
I think it was BTC Breakdown newsletter, or Bitcoin Minimalist
